cfme/tests/infrastructure/test_provisioning_dialog.py

```diff
@@ -293,3 +293,36 @@ def test_provisioning_schedule(provisioner, provider, prov_data, vm_name):
     template_name = provider.data['provisioning']['template']
 
     provisioner(template_name, prov_data, delayed=provision_time)
+
+
+@pytest.mark.rhv2
+@pytest.mark.uncollectif(lambda provider: not provider.one_of(RHEVMProvider),
+                         reason='These profile names are RHV specific.')
+@pytest.mark.parametrize('vnic_profile', ['<No Profile>', '<Use template nics>'])
+def test_provisioning_default_vnic_profiles(provisioner, provider, prov_data,
```

Comment:
I think name "test_provisioning_default_vnic_profiles" should not contain "default", since this test is for default (<'<No Profile>'), but also  for '<Use template nics>'.

Reply:
Agreed